Gemfile.lock

```diff
@@ -129,6 +129,7 @@ GEM
     faraday_middleware (0.12.2)
       faraday (>= 0.7.4, < 1.0)
     ffi (1.14.2)
+    ffi (1.14.2-x64-mingw32)
```

Comment:
why did you include this?

Reply:
That was added in automatically when I did the bundle install.

Reply:
Can you do a git checkout to reset the Gemfile.lock and run bundle install again with the removal of that line above, I think then it will remove this change.